packages/wrangler/src/hyperdrive/index.ts

```diff
@@ -159,15 +159,21 @@ export function getOriginFromArgs<
 			(url.protocol == "postgresql:" || url.protocol == "postgres:")
 		) {
 			url.port = "5432";
+		} else if (url.port === "" && url.protocol == "mysql:") {
+			url.port = "3306";
 		}
 
 		if (url.protocol === "") {
 			throw new UserError(
-				"You must specify the database protocol - e.g. 'postgresql'."
+				"You must specify the database protocol - e.g. 'postgresql'/'mysql'."
 			);
-		} else if (url.protocol !== "postgresql:" && url.protocol !== "postgres:") {
+		} else if (
+			url.protocol !== "postgresql:" &&
```

Comment:
Could be nice to use `url.protocol.startsWith` to handle the colon and also consider normalizing the case here since we're displaying the schemes with the "correct" mixed case.